content/blog/building-a-future-of-cloud-engineering/index.md

```diff
@@ -83,18 +83,18 @@ approach.
 We believe in a future where developers and operators alike can meet on
 common ground to work together. We like to call this combined discipline
 **cloud engineering**. It's not Dev, it's not DevOps, and it's not
-SRE; it's the best of all of them. In this new future, our teams will
+SRE; cloud engineering is the best of all of them. In this new future, our teams will
 experience more joy, collaboration, and productivity than ever before.
 
 Enter Pulumi.
 
 Using your favorite general purpose programming language -- whether it
 is JavaScript, TypeScript, Python, Go, or something else -- you can
-express the entirety of your cloud software. You can develop it in your
+express the entirety of your cloud software. As a cloud engineer, you can develop it in your
```

Comment:
This paragraph doesn't quite work as well with these additions, IMO.

Reply:
Research was showing that we were a little bit under average for variations of "cloud engineer/ing" compared with our SERP competitors. This seemed a natural place to add it in, so it'd be great if we could keep even just a couple instances, or even just the first one.